tango/integrations/torch/train.py

```diff
@@ -484,38 +481,35 @@ def _train(
 
     def is_best_checkpoint() -> bool:
         """
-        A closure that we'll call when saving checkpoints to check if we should hardlink
+        A closure that we'll call when saving checkpoints to check if we should link
```

Comment:
To be sure, this has to be a hardlink, because the checkpoint that's saved under a name that derives from the step count might go away.

I guess we could do it the other way, where the current best checkpoint becomes a real file, with a symlink pointing to it. Then it doesn't have to be a hardlink anymore. But either way it doesn't work on Windows, so why bother?